maat-court-data-api/src/main/java/gov/uk/courtdata/repository/FinancialAssessmentDetailsRepository.java

```diff
@@ -2,9 +2,11 @@
 
 import gov.uk.courtdata.entity.FinancialAssessmentDetailEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
 import java.util.List;
 
+@Repository
```

Comment:
I'm guessing this is just for consistency with the other classes? Technically its not required when the class extends `JpaRepository`.

Reply:
Yep, there is also a pointcut in the x-ray spring aop library that checks for the annotation before it enables tracing on repositories.